datafusion/core/tests/sql/sql_api.rs

```diff
@@ -19,6 +19,23 @@ use datafusion::prelude::*;
 
 use tempfile::TempDir;
 
+#[tokio::test]
+async fn test_window_function() {
+    let ctx = SessionContext::new();
+    let df = ctx
+        .sql(
+            r#"SELECT
+        t1.v1,
+        SUM(t1.v1) OVER w + 1
+        FROM
+        generate_series(1, 10000) AS t1(v1)
+        WINDOW
```

Comment:
I think we should at least have a .slt test that shows this query running and producing the same result as postgres, perhaps with a smaller number of series:

```sql
postgres=# SELECT
  t1.v1,
  SUM(t1.v1) OVER w
FROM
  generate_series(1, 5) AS t1(v1)
WINDOW
  w AS (ORDER BY t1.v1 ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW);
 v1 | sum
----+-----
  1 |   1
  2 |   3
  3 |   6
  4 |  10
  5 |  15
(5 rows)
```